pages/top_rated/index.tsx

```diff
@@ -1,18 +1,34 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 
 import TopRatedMovie from '../../src/components/TopRatedMovie/TopRatedMovie';
 import { moviesService } from '../../src/services/movies.service';
+import { addToFavorites } from '../../src/store/favoritesSlice';
+import { RootState } from '../../src/store/store';
+
 import { Movie } from '../../types';
 
 type TopMovieProps = {
   topMovie: Movie[];
 };
 
 const TopRated = ({ topMovie }: TopMovieProps) => {
```

Comment:
you are inconsistent with your naming here. It's going to be confusing if you are going to reuse this type and component somewhere in the future. Either rename to `TopRatedMovies` and `TopRatedMoviesProps` or `TopRated` and `TopRatedProps`, first way is more preferable